### ISSUE TITLE
C compiler warning: 'ssl3_ciphers' redeclared without dllimport attribute: previous dllimport ignored

### ISSUE THREAD

**Issue by vszakats** [openssl/openssl#673]
(using mingw 5.3.0)
```
gcc -DOPENSSL_THREADS -DOPENSSL_PIC -DOPENSSL_BN_ASM_PART_WORDS -DOPENSSL_IA32_SSE2 -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_GF2m -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DMD5_ASM -DRMD160_ASM -DAES_ASM -DVPAES_ASM -DGHASH_ASM -DECP_NISTZ256_ASM -DPOLY1305_ASM -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines\"" -D_WINDLL  -D_MT -DL_ENDIAN -DWIN32_LEAN_AND_MEAN -DUNICODE -D_UNICODE -m32 -Wall -O3 -fomit-frame-pointer -DWHIRLPOOL_ASM -fno-ident -Iinclude  -I. -c -o ssl/s3_lib.o ssl/s3_lib.c
ssl/s3_lib.c:162:33: warning: 'ssl3_ciphers' redeclared without dllimport attribute: previous dllimport ignored [-Wattributes]
 OPENSSL_GLOBAL const SSL_CIPHER ssl3_ciphers[] = {
                                 ^
```

Not sure what the fix is, but here are the contradicting lines:
```
ssl/s3_lib.c:OPENSSL_GLOBAL const SSL_CIPHER ssl3_ciphers[] = {
ssl/s3_lib.c:        return (&(ssl3_ciphers[SSL3_NUM_CIPHERS - 1 - u]));
ssl/s3_lib.c:    cp = OBJ_bsearch_ssl_cipher_id(&c, ssl3_ciphers, SSL3_NUM_CIPHERS);
ssl/ssl_locl.h:OPENSSL_EXTERN const SSL_CIPHER ssl3_ciphers[];
```

Maybe the best would be not to directly expose this variable to the outside world.

**Comment by levitte** [comment]
O.o

That array can be made static...  that'll avoid the whole problem.

**Comment by vszakats** [comment]
I'll make it a PR.

**Comment by levitte** [comment]
Sure

**Comment by vszakats** [comment]
Fixed in f0ff328e360f56d8a79cbb61b2931a13fb7199c2.
